INSTALL.md

```diff
@@ -49,20 +60,24 @@ sudo apt-get install -y libbpf-dev
 ### Install libbpf from source
 
 ```bash
-https_proxy="https://user:password@proxy.server:port" git clone https://github.com/libbpf/libbpf.git
+git clone https://github.com/libbpf/libbpf.git
```

Comment:
Should create a new section at the top of the file to mention the https_proxy is required if you are downloading from the internet. If you're behind a proxy, most people know they need to configure the proxy, so this would just be for completeness.

Reply:
I am starting to think this proxy statement is not required, if the developer is behind a proxy they had better know how to use it. We should not be trying to teach them how to setup a proxy.

Reply:
I agree.